### PR TITLE
Move block server-side logic into same directory as client-side logic

### PR DESCRIPTION
**Pull request by westonruter**
When working with dynamic blocks it is not ideal for the JS and PHP logic to be separated out in two separate locations. Ideally the dynamic block server rendering logic as well as unit tests could be placed right inside of the same directory as the client logic. 

This would also impact the Latest Comments block: #1931.

### CODE REVIEW

**Review by aduth — CHANGES_REQUESTED** [review]
See also: #1095

We need to remove this line:

https://github.com/WordPress/gutenberg/blob/44ad46a/bin/build-plugin-zip.sh#L71

And ensure that block PHP files are included in the generated plugin.

**Comment by westonruter** [comment]
@aduth thanks for that. Fixed in https://github.com/WordPress/gutenberg/pull/2014/commits/bfe31c27e731615c1c3d9c7e8fd00c6fb8b17b74

File: gutenberg.php [hunk 1]
```diff
@@ -25,6 +25,7 @@
 	require_once dirname( __FILE__ ) . '/lib/parser.php';
 	require_once dirname( __FILE__ ) . '/lib/register.php';
 
-	// Register server-side code for individual blocks.
```
Comment (aduth) [inline gutenberg.php]: Should we preserve this comment?

**Review by aduth — APPROVED** [review]
Verified package plugin loads block correctly.
Verified `composer lint` surfaces issues in the block PHP file.

**Pull request merged**